phpunit.xml.dist

```diff
@@ -12,23 +12,26 @@
          bootstrap="Tests/bootstrap.php"
 >
     <testsuites>
-        <testsuite name="User Bundle Test Suite">
-            <directory>./Tests/</directory>
+        <testsuite name="UserBundle Test Suite">
```

Comment:
"UserBundle" isn't a thing, I think it was better before. "SonataUserBundle" is our user bundle.